imageGal.html

```diff
@@ -1,62 +1,101 @@
-
-<!DOCTYPE html>
 <html>
-    <head>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="imageGal1.css">
-    </head>
-    <body onload="defaultImageDisplay()">
-
-        <div class = "mainGallery">
-
-            <div class ="subGallery" >
-                <div class="c">
-                    <img id="defImage" src="images/capture.jpg" alt="Paris" width="290px" height="240px" onclick="myFunction(this);" >
-                </div>
-            </div>
-
-            <div class = "subGallery" >
-                <div class="c">
-                    <img class="otherImages" src="images/coconutTree.jpg" alt="Paris" width="290px" height="200px" onclick="myFunction(this);" >
-                </div>
-            </div>
-
-            <div class ="subGallery" >
-                <div class="c">
-                    <img class="otherImages" src="images/landscape.jpg" alt="Paris" width="290px" height="240px" onclick="myFunction(this);" >
-                </div>
-            </div>
-
-            <div class = "subGallery" >
-                <div class="c">
-                    <img class="otherImages" src="images/butterfly.jpg" alt="Paris" width="290px" height="200px" onclick="myFunction(this);" >
-                </div>
-            </div>
-
+  <head> 
+      <link rel="stylesheet" href="imageGal1.css">
+      <script src="imageGal3.js"></script>
```

Comment:
This script tag must be add just before body closing tab. Else when the script executes it will not find your` <div id="mainGallery" >` or `<div class="sub_container">`. After you have done this you can remove the script tag with direct code you have used below